assets/section-image-banner.css

```diff
@@ -197,8 +197,13 @@
 }
 
 .banner__box {
-  border: 0;
-  padding: 4rem 3.5rem;
+  border: var(--text-boxes-border-width) solid rgba(var(--color-foreground), var(--text-boxes-border-opacity));
+  padding: var(--text-boxes-padding);
```

Comment:
This is where trying to use a single pixel value rather than magic sizes to control multiple sections is going to be tricky. Or at least require a larger range. With banner_box previously designed at ~4rem, the current max for text boxes being 2rem is going to slim down those designs.

Reply:
I agree with Ken. The Multicolumn section has some inner padding right now that doesn't allow the border to ever touch the text, while the banner section doesn't. 

The border is very close to the text and touches the button(s) if they're present. 
I think we'll need at least a little bit of default padding. 

<img width="1162" alt="Screen Shot 2021-11-16 at 2 03 25 PM" src="https://user-images.githubusercontent.com/26778637/142048996-ac939518-0f66-425e-99ad-b64ccd4b3946.png">

Same for the password page:
<img width="884" alt="Screen Shot 2021-11-16 at 2 15 57 PM" src="https://user-images.githubusercontent.com/26778637/142050703-3a419bbc-e974-45d9-bb81-cb9e260ef4ad.png">

Reply:
An alternative would be for it to be allowed to touch but to add `overflow: hidden` so that a merchant could create a text box with the buttons flush with the border - it could be an effect they're looking for.